CalcIgor.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CourseApp

```

Comment:
Rename namespace and project into CalculatorSolution or something like that

Reply:
This one was at the start - and changing namespace is good idea, but as this one task was created at the start - optional